packages/cli/src/index.ts

```diff
@@ -142,7 +147,16 @@ async function main(config: ParsedConfig, isWatchMode: boolean) {
         .on('add', cb)
         .on('change', cb);
     } else {
-      const fileList = glob.sync(pattern);
+      /**
+       * If the user didn't provide the -f paramter, we're using the list of files we got from glob.
+       * If he did, we're using glob file list to detect if his provided file should be used with this transform.
+       */
+      const globList = glob.sync(pattern);
+      const fileList = fileOverride
+        ? globList.includes(fileOverride)
+          ? [fileOverride]
+          : []
+        : globList;
```

Comment:
Lets normalize ternaries here:
```suggestion
      let fileList = glob.sync(pattern);
      if (fileOverride) {
        fileList = fileList.includes(fileOverride) ? [fileOverride] : [];
      }
```